src/ApiPort.Vsix/ApiPort.Vsix.sln

```diff
@@ -0,0 +1,50 @@
+﻿
```

Comment:
Sounds good. I just saw that file and was wondering what its use was... I've removed it now.